### DISCUSSION TITLE
OC Suite Configurator Guide

### DISCUSSION THREAD

**Discussion by ChanseyIsTheBest** [hanai3Bi/Switch-OC-Suite#43] [Ram Timing | Customization]
Outdated use these instead
https://github.com/hanai3Bi/Switch-OC-Suite/discussions/56

**Comment by jang2ah2** [comment]
With my Erista Ram (samsung k4f6e304hb-mgch) 
Tried multiple timing to make 1996Mhz works with EMC Vddq set to 1175mv. And I make a quick test playing MarioKart, and It keeps crashing not even completing the 1st Lap. 
And when I tried to set **EMC Vddq 1250mv** and set **Ram to 1996Mhz** and **everything else is unchange**
Playing MarioKart for almost 2 hours now and some other games with no crashes. 

I'm not sure though if its worth the risk to set Ram speed from 1862Mhz to 1996Mhz with EMC Vddq set to max (1250mv)

**Reply by ChanseyIsTheBest** [reply]
Personally I wouldn't above 1200mv. If 1996Mhz isn't stable at that I would go back to 1862,Mhz especially if 1862Mhz needs no OV. I believe the safe chip max is 1175mv. People seem to like testing stability on Borderlands 3 and Borderlands Pre-Sequel and Nier Automata because they seem very prone to crashing due to memory corruption.
